core/src/test/java/org/elasticsearch/action/admin/cluster/node/tasks/TestTaskPlugin.java

```diff
@@ -226,7 +226,7 @@ protected NodesResponse newResponse(NodesRequest request, List<NodeResponse> res
         protected String[] filterNodeIds(DiscoveryNodes nodes, String[] nodesIds) {
             List<String> list = new ArrayList<>();
             for (String node : nodesIds) {
-                if (nodes.getDataNodes().containsKey(node)) {
+                if (nodes.nodeExists(node)) {
```

Comment:
this as well was a valid change even before? Trying to figure out what the implications are...